features/completed-form-available-to-download.feature

```diff
@@ -0,0 +1,58 @@
+@e2e
+@javascript
+Feature:
+  As an ATOS admin
+  I want to be able to download completed forms
+  So I can triage an employees claim against their employer
+
+  Scenario: PDF format
+    Given I am making a claim
+    When I submit a completed Employment Tribunal form
+    Then the completed form in PDF format is available for ATOS to download and validate
+
+  Scenario: TXT format
+    Given I am making a claim
+    When I submit a completed Employment Tribunal form
+    Then the completed form in TXT format is available for ATOS to download and validate
+
+  Scenario: XML format
+    Given I am making a claim
+    When I submit a completed Employment Tribunal form
+    Then the completed form in XML format is available for ATOS to download and validate
+
+  Scenario: No representative
+    Given I am making a claim with no respresentative
+    When I submit a completed Employment Tribunal form
+    Then the completed form in PDF format is available for ATOS to download and validate
+
+  Scenario: One other person making claim
+    Given I am making a claim with one other person
+    When I submit a completed Employment Tribunal form
+    Then the completed form in PDF format is available for ATOS to download and validate
+
+  Scenario: Validate TXT file when uploading CSV data with 7 other people making the claim
+    Given I am making a claim with 7 other people
```

Comment:
You can make a claim with multiple people using either the web form (limited number) or using csv. Maybe it is worth adding some wording in this setup step to suggest that you are using csv here ?

Reply:
Is this OK? 

 Scenario: Validate TXT file when uploading CSV data
    Given 7 employees making a claim by uploading CSV file
    When the completed form is submitted
    Then I can download the uploaded CSV data in TXT format

Reply:
Yeah that reads better I think